client/onboarding-prototype/types.ts

```diff
@@ -17,4 +17,18 @@ export type OnboardingFields = {
 	go_live_timeframe?: string;
 };
 
-export type OnboardingSteps = 'personal' | 'business' | 'store';
+export interface EligibleResult {
+	result: string;
```

Comment:
This definition is more accurate and will improve autocompletion.

```suggestion
	result: 'eligible' | 'not_eligible';
```

Reply:
Oh, right, forgot about it! Thanks, fixed in https://github.com/Automattic/woocommerce-payments/pull/5779/commits/ecca78b61d677e1e71fa3cbd83591c3046d3a87d.